lib/signs/actions/signs.js

```diff
@@ -50,36 +51,22 @@ export type SignActions = ActionType<typeof receivedGtfsEntities> |
 let nextSignId = 0
```

Comment:
Not related to this id thing, but the `receivedRtdSigns` action is being called the wrong way and destructured the wrong way in the reducer.

Reply:
I'm pretty sure the signs module is all but deprecated, but I yea will fix this.